scale/models/scale_scale.py

```diff
@@ -14,10 +15,14 @@
 class Scale(models.Model):
     _name = 'scale.scale'
     _description = "Scale"
+    _sql_constraints = [
+        ("name_uniq", "UNIQUE(name)", _("Name must be unique!")),
+    ]
 
     name = fields.Char(required=True)
-    ip = fields.Char(required=True)
-    port = fields.Integer(required=True)
+    ip = fields.Char()
+    port = fields.Integer()
+    url = fields.Char()
```

Comment:
Add a proper label, in this case (otherwise, Odoo will put "Url"):
```suggestion
    url = fields.Char(string="URL")
```